src/cromshell/status/command.py

```diff
@@ -0,0 +1,144 @@
+    workflow_status = workflow_status_description["status"]
+
+    # Set return value based on workflow status
+    if workflow_status in ("Failed", "Aborted", "fail"):
+        ret_val = 1
+        log.display_logo(io_utils.dead_turtle)
+    elif workflow_status == "Succeeded":  # change to Running for the final version.
+        # Status claims this workflow is running fine, but we need to check to see
+        # if there are any failed sub-processes.
+        # To do this, we use the `execution-status-count` logic with some filtering:
+        # TODO : Use this as a template for the Metadata subcommand
+        # Get execution status count and filter the metadata down:
+        request_meta_out = requests.get(
+            f"{config.cromwell_server}{config.api_string}{workflow_id}/metadata?{config.slim_metadata_parameters}"
+        )
+
+        # tmp_metadata holds the workflow metadata as a dictionary
+        tmp_metadata = json.loads(request_meta_out.content.decode("utf-8"))
+        execution_status_count = get_metadata_status_summary(tmp_metadata)
+
+        # Check for failure states:
+        failed = False
+        for task_status in execution_status_count:
+            for key in task_status:
+                if "Failed" in str(task_status[key]):
+                    failed = True
+                    break
+
+        # Check for failures:
+        if not failed:
+            # We could not find 'Fail' in our metadata, so our
+            # original Running status is correct.
+            log.display_logo(io_utils.turtle)
+        else:
+            log.display_logo(io_utils.doomed_logo)
+            workflow_status = "DOOMED"
+            message = (
+                "The workflow is Running but one of the instances "
+                "has failed which will lead to failure."
+            )
+            requested_status_json = f'{{"status":"{workflow_status}","id":"{workflow_id}"}}\n{message}'
+
+    else:
+        log.display_logo(io_utils.turtle)
+
+    # Display status to user:
+    line_string = requested_status_json
+    print(line_string.replace(",", ",\n"))
+
+    # Update ${CROMWELL_SUBMISSIONS_FILE}:
+    with fileinput.FileInput(
+             config.submission_file, inplace=True, backup=".bak"
+    ) as csv_file:
+        reader = csv.DictReader(csv_file, delimiter="\t")
+        print("\t".join(reader.fieldnames))
+        for row in reader:
+            if row["CROMWELL_SERVER"] == config.cromwell_server and row["RUN_ID"] == workflow_id:
+                row["STATUS"] = workflow_status
+                print("\t".join(x for x in row.values() if x))
+            else:
+                print("\t".join(x for x in row.values() if x))
+
+    return ret_val
+
+
+def get_metadata_status_summary(workflow_metadata):
+    """Get the status for each call in a workflow and the frequency of those statuses"""
+    # workflow_metadata holds the workflow metadata as a dictionary
+    workflow_status_count = []
```

Comment:
```suggestion
```

You don't seem to need this variable.  See below for another change that with this one will eliminate it.